molecule/cookiecutter/scenario/driver/ec2/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -8,8 +8,9 @@
   vars:
     ssh_user: ubuntu
     ssh_port: 22
+    default_region: "{{ lookup('env', 'AWS_DEFAULT_REGION') | default('us-east-1', true) }}"
```

Comment:
That is an essential feature, especially as my default region is UK.

Reply:
This line is following default AWS behavior
> When you use an endpoint with no Region, AWS routes the Amazon EC2 request to US East (N. Virginia) (us-east-1), which is the default Region for API calls.

doc - https://docs.aws.amazon.com/general/latest/gr/rande.html